.drone.yml

```diff
@@ -0,0 +1,83 @@
+---
+kind: pipeline
+name: default
+
+services:
+  - name: docker
+    image: docker:19.03-dind
+    entrypoint:
+      - dockerd
+    command:
+      - --dns=8.8.8.8
+      - --dns=8.8.4.4
+      - --mtu=1440
+      - --log-level=error
+    privileged: true
+    volumes:
+      - name: docker-socket
+        path: /var/run
+
+steps:
+  - name: build-pull-request
+    image: autonomy/build-container:latest
+    pull: always
+    commands:
+      - docker buildx create --driver docker-container --platform linux/amd64 --name local --use
+      - bldr pack --progress=plain -f pkg.yaml
```

Comment:
this version without `-o` requires new `bldr`, I can keep `-o autonomy` still though.. 

but it allows to change the organization with just change to bldr